include/s3select_oper.h

```diff
@@ -315,7 +315,7 @@ class value
 			            m_to_string.assign( boost::lexical_cast<std::string>(__val.num) );
                 }else if(type == value_En_t::FLOAT){
                         m_to_string = std::to_string(__val.dbl);
-                }else {
+                }else if (type == value_En_t::TIMESTAMP) {
```

Comment:
don't think it's needed , since there are 4 types int,float,timestamp and string.
that else is possible for timestamp only.
if that's wrong (not default) , should be a test-case for that.

Reply:
without this fix, the following line is crashing:
https://github.com/galsalomon66/s3select/pull/5/files#diff-6a7cb2c9952c4063a84ffeb72d918e38R417
seems like the combination of a object name with a dot in it, and using a function